imas_validator/assets/rulesets/generic/ggd.py

```diff
@@ -4,29 +4,31 @@
 from imas.ids_data_type import IDSDataType
 from imas.ids_defs import IDS_TIME_MODE_HETEROGENEOUS, IDS_TIME_MODE_HOMOGENEOUS
 
-SUPPORTED_IDS_NAMES = [
-    "edge_profiles",
-    "edge_sources",
-    "edge_transport",
-    "mhd",
-    "radiation",
-    "runaway_electrons",
-    "plasma_profiles",
-    "plasma_sources",
-    "plasma_transport",
-]
-
-# TODO: Some IDSs do not have the grid structure in a separate `grid_ggd` object, as
-# described by the GGD guidelines. They are currently not covered by the validations.
-# If the DD for these IDSs stays like this, they will need to be handled separately.
-# GGD grid locations for each IDS:
-# wall (description_ggd/grid_ggd)
-# equilibrium (grids_ggd/grid)
-# distribution_sources (source/ggd/grid)
-# distributions (distribution/ggd/grid)
-# tf (field_map/grid)
-# transport_solver_numerics (boundary_conditions_ggd/grid)
-# waves (coherent_wave/full_wave/grid)
+# As IDSs store their GGD grid and GGD AoS in different locations, they are explicitly
+# mapped here. If you want to enable GGD validation for a new IDS, amend it to this
+# mapping. This takes the following form:
+# "<IDS name>": ("<GGD grid IDS path>", "<GGD IDS path>")
+GGD_PATHS_PER_IDS = {
+    "edge_profiles": ("grid_ggd", "ggd"),
+    "edge_sources": ("grid_ggd", "source/ggd"),
+    "edge_transport": ("grid_ggd", "model/ggd"),
+    "mhd": ("grid_ggd", "ggd"),
+    "radiation": ("grid_ggd", "process/ggd"),
+    "runaway_electrons": ("grid_ggd", "ggd_fluid"),
+    "plasma_profiles": ("grid_ggd", "ggd"),
+    "plasma_sources": ("grid_ggd", "source/ggd"),
+    "plasma_transport": ("grid_ggd", "model/ggd"),
+    "wall": ("description_ggd", "description_ggd/ggd"),
+    "equilibrium": ("grids_ggd/grid", "time_slice/ggd"),
+    "distribution_sources": ("source/ggd/grid", "source/ggd"),
+    "distributions": ("distribution/ggd/grid", "distribution/ggd"),
+    "tf": ("field_map/grid", ""),  # doesn't have GGD?
```

Comment:
Looks like the field_map itself contains quantities defined on the grid. Even though it's not called `ggd` we should probably validate these?

```suggestion
    "tf": ("field_map/grid", "field_map"),
```